docs/install/tutorials/provision.rst

```diff
@@ -0,0 +1,148 @@
+:orphan:
+.. _provision:
+
+Hard drive provisioning of multiple Kolibri servers
+===================================================
```

Comment:
Have you seen this already? https://kolibri.readthedocs.io/en/latest/install/tutorials/rpi.html#saving-your-image-for-replication

Not sure if useful for this guide, but I added `provisiondevice` in that guide.

I guess the Raspberry Pi tutorial could perhaps cross-reference to this one? I.e. "For more general advice, read ..."

Since this is also a tutorial, should we kick off the Tutorial section? :)

"Provision" is not a normal word in my vocabulary, so would also perhaps suggest to find an alternative for the headline. "Copying a Kolibri setup for re-use" or similar.

Reply:
Cross-referenced to RPi tutorial, and put back `Tutorials` in the TOC :tada: